server/routes/manageConditions/handlers/additionalLicenceConditionInput.ts

```diff
@@ -42,20 +42,9 @@ export default class AdditionalLicenceConditionInputRoutes {
     const { licenceId } = req.params
     const { conditionId } = req.params
     const { user, licence } = res.locals
-    const isFileUploadRequest = req.file && req.file.fieldname === 'outOfBoundFilename'
```

Comment:
one big benefit of this approach is that we don't need to have logic like this or `'outOfBoundArea' in req.body`